client/components/RatingsReviews/RatingsReviews.jsx

```diff
@@ -5,19 +5,44 @@ import RatingBreakdown from './RatingBreakdown.jsx';
 import AddReview from './AddReview.jsx';
 import ProductBreakdown from './ProductBreakdown.jsx';
 import SortReviews from './SortReviews.jsx';
+import axios from 'axios';
+
+const getMetaData = function (productId) {
+  let url = 'http://localhost:3000/proxy/api/fec2/hratx/reviews/meta?product_id=' + productId;
+  return axios.get(url)
+    .then((response) => {
+      return response;
+    })
+    .catch((err) => {
+      throw err;
+    });
+};
 
 const RatingsAndReviews = (props) => {
-  return (<div className='jcontainer'>
-    <SortReviews />
-    <ReviewList id={props.reviewId || 12012}/>
-    <div className='ReviewSideBar'>
-      <div className='SideBarContainer'>
-        <RatingSummary />
-        <RatingBreakdown />
-        <ProductBreakdown />
+  const [metaData, setMetaData] = useState([]);
+  if (metaData.length === 0) {
+    getMetaData(props.product_id)
+      .then((data) => {
+        setMetaData(data);
```

Comment:
rather than setting metaData to the entire AJAX response, maybe only set it with the data portion. (by your naming convention, data.data would be just the data. consider changing the naming convention as well)